ui/pages/onboarding-flow/creation-successful/creation-successful.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import reactRouterDom from 'react-router-dom';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import {
+  ONBOARDING_PIN_EXTENSION_ROUTE,
+  ONBOARDING_PRIVACY_SETTINGS_ROUTE,
+} from '../../../helpers/constants/routes';
+import CreationSuccessful from './creation-successful';
+
+describe('Creation Successful Onboarding View', () => {
+  const useHistoryOriginal = reactRouterDom.useHistory;
+  const pushMock = jest.fn();
+  beforeAll(() => {
+    jest
+      .spyOn(reactRouterDom, 'useHistory')
+      .mockImplementation()
+      .mockReturnValue({ push: pushMock });
+  });
+  afterAll(() => {
+    reactRouterDom.useHistory = useHistoryOriginal;
```

Comment:
Do you have to do this? My understanding is that Jest mocks automatically reset each test.

Reply:
That's right! Will remove